Formula/v/vcprompt.rb

```diff
@@ -1,15 +1,10 @@
 class Vcprompt < Formula
   desc "Provide version control info in shell prompts"
-  homepage "https://hg.gerg.ca/vcprompt"
-  url "https://hg.gerg.ca/vcprompt/archive/1.2.1.tar.gz"
-  sha256 "fdf26566e2bd73cf734b7228f78c09a0f53d0166662fcd482a076ed01c9dbe36"
+  homepage "https://github.com/wfr/vcprompt"
+  url "https://github.com/wfr/vcprompt/archive/refs/tags/1.2.1.tar.gz"
+  sha256 "6fd44d453c59efcfb9e01e6fe3a47573d32a64b81a89c19df7bee84f12a2c5be"
```

Comment:
Arch uses https://github.com/powerman/vcprompt for this, and everyone else uses the old URL. How did you get this new source?